pkg/controller/controller_binding.go

```diff
@@ -180,8 +180,8 @@ func (c *controller) reconcileServiceInstanceCredential(binding *v1alpha1.Servic
 			"ServiceInstanceCredential \"%s/%s\" references a non-bindable ServiceClass (%q) and Plan (%q) combination",
```

Comment:
I think we should have both the k8s name and external name in this log message now.

Reply:
I think there will probably be a number of log message that now need to have clarification about k8s names and external names, so I would be happy to do those in a follow-up

Reply:
Ack. ok on followup. Created:
https://github.com/kubernetes-incubator/service-catalog/issues/1279